.gitignore

```diff
@@ -73,7 +73,6 @@ web_modules/
 .yarn-integrity
 
 # dotenv environment variable files
-.env
```

Comment:
I think this was a holdover removal from experimentation. I don't see a `.env` file so we should add it back to this file.

Reply:
We could, but I kept it because [the NextJS docs recommend](https://nextjs.org/docs/app/building-your-application/configuring/environment-variables) allowing them to be committed.

> Good to know: .env, .env.development, and .env.production files should be included in your repository as they define defaults. .env*.local should be added to .gitignore, as those files are intended to be ignored. .env.local is where secrets can be stored.

Reply:
Oh man, that advice is going to get a lot of folks in trouble 🙈 I've always followed the guidance not to check in a .env file.

Reply:
Yeh that's better advice. I've reverted this.